ansible/roles/prometheus/templates/prometheus.yml.j2

```diff
@@ -151,9 +151,30 @@ scrape_configs:
       key_file: "{{ prometheus_ssl_dir }}/{{ inventory_hostname }}.key"
     static_configs:
       - targets:
-{% for host in (groups.have_netdata) %}
+{% for host in (groups.have_netdata|sort) %}
         - {{ host }}:9100
 {% endfor %}
+        - ams-pg.ooni.nu:9100
+
+  - job_name: 'raw-netdata'
+    scrape_interval: 5s
+    scheme: http
+    metrics_path: "/api/v1/allmetrics?format=prometheus&help=yes"
+    params:
+      format: [prometheus]
+    static_configs:
+      - targets:
+        - ams-pg.ooni.nu:19999
+
+  - job_name: 'raw postgres-exporter'
+    scrape_interval: 5s
+    scheme: https
+    metrics_path: /api/v1/allmetrics
+    params:
+      format: [prometheus]
+    static_configs:
+      - targets:
+        - ams-pg.ooni.nu:9187
```

Comment:
What is the difference between these two netadata endpoints and the ones that have_netdata?